### PR TITLE
Translation/billing

### LINKED ISSUE #280: Localization for Billing Module
Now that i18n support has been added to HospitalRun via PR #248 and issue #141 we need to localize the app.  This issue is to track localization for the billing module.  Localization in HospitalRun uses [Ember-I18n](https://github.com/jamesarosen/ember-i18n).  The translations/default text will be placed in app/locales/en/translations.js.
See https://github.com/jamesarosen/ember-i18n/wiki/Doc:-Translating-Text for information on how to localize the templates.

### PR DESCRIPTION
**Pull request by mellisdesigns**
Fixes #280.

**Changes proposed in this pull request:**
- Translations for all Billing Routes

Localisation of the complete billing routes including invoicing and pricing. I have tried my best to organise what I think to be generic translations and specific translations. A billing localisation object exists to add any text that is very specific to that module. Happy to amend any discussions about the current grouping I have done in this PR.

*Note: pull requests without proper descriptions may simply be closed without further discussion. We appreciate your contributions, but need to know what you are offering in clearly described format. Thanks! (you can delete this text)*

cc @HospitalRun/core-maintainers

### CODE REVIEW

File: app/locales/en/translations.js [hunk 1]
```diff
@@ -279,7 +279,24 @@ export default {
     physician: 'Physician',
     assisting: 'Assisting',
     anesthesia: 'Anesthesia',
-    procedures: 'Procedures'
+    procedures: 'Procedures',
+    number: 'Number',
+    bill_date: 'Bill Date',
```
Comment (billybonks) [inline app/locales/en/translations.js]: i think quite a few of these are actually billing section only, shared between prices and invoices. can we nest those under billing.labels?

**Comment by billybonks** [comment]
Need to translate pricing route as well. to complete issue, but would be happy to merge this first after the comment is addressed

**Comment by billybonks** [comment]
@mellisdesigns @jkleinsc we can close this i incorporated changes in https://github.com/HospitalRun/hospitalrun-frontend/pull/515

**Comment by jkleinsc** [comment]
ok thanks @billybonks.  I'll close this.

**Pull request closed without merging**